src/main/kotlin/com/exactpro/th2/codec/oracle/logminer/LogMinerTransformer.kt

```diff
@@ -116,7 +121,9 @@ class LogMinerTransformer(private val config: LogMinerConfiguration) : IPipeline
         }
 
         return message is ParsedMessage
-                && (message.protocol.isBlank() || TransformerFactory.PROTOCOLS.contains(message.protocol))
+                && (message.protocol.isBlank() || TransformerFactory.PROTOCOLS.contains(message.protocol)).also {
+            LOGGER.debug { "The ${message.id.logId} message isn't appropriate, type: '${message::class.java}', protocol: '${message.protocol}'" }
```

Comment:
I believe you should not do that here. If you want to log that the message is not "appropriate" it should be done in the place where `isAppropriate` method is called. Not inside the method itself

Reply:
I would like to place here to be closely to the check logic

Reply:
Up to you)